src/LiveComponent/README.md

```diff
@@ -1020,35 +996,27 @@ First use the `ValidatableComponentTrait` and add any constraints you need:
 
 ```php
 use App\Entity\User;
+use Symfony\UX\LiveComponent\Attribute\AsLiveComponent;
 use Symfony\UX\LiveComponent\Attribute\LiveProp;
-use Symfony\UX\LiveComponent\LiveComponentInterface;
 use Symfony\UX\LiveComponent\ValidatableComponentTrait;
 use Symfony\Component\Validator\Constraints as Assert;
 
-class EditUserComponent implements LiveComponentInterface
+#[AsLiveComponent('edit_user')]
+class EditUserComponent
 {
     use ValidatableComponentTrait;
 
-    /**
-     * @LiveProp(exposed={"email", "plainPassword"})
-     * @Assert\Valid()
-     */
+    #[LiveProp(exposed: ['email', 'plainPassword'])]
+    #[Assert\Valid]
     public User $user;
 
-    /**
-     * @LiveProp()
-     * @Assert\IsTrue()
-     */
+     #[LiveProp]
+     #[Assert\IsTrue]
     public bool $agreeToTerms = false;
-
-    public static function getComponentName() : string
-    {
-        return 'edit_user';
-    }
 }
 ```
 
-Be sure to add the `@Assert\IsValid` to any property where you want
+Be sure to add the `Assert\IsValid` to any property where you want
```

Comment:
Is this also available as a PHP8 attribute? Maybe clearify it by writing `... add the #[Assert\IsValid] attribute to any ...`